components/Alert/badge/badge.css

```diff
@@ -0,0 +1,66 @@
+
+  .icons {
+    height: 5rem;
+  }
+  
+  .number {
+    position: absolute;
+    background-color: var(--primary);
+    color: var(--white);
+    padding: .5rem;
```

Comment:
.5 rem = 8px , use px for less than equal to 10px

Reply:
ok thanks